### DISCUSSION TITLE
Lesson 20: --fork-url not provided error

### DISCUSSION THREAD

**Discussion by 0xmihirsahu** [Cyfrin/foundry-full-course-cu#1789] [Q&A] [answered]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/114948140/54592b3b-2514-4893-b81f-72b306831b8a)

**Comment by urosognjenovic** [comment, marked as answer]
There's a typo in your command. 

You need ```$SEPOLIA_RPC_URL``` instead of ```$SEPLOIA_RPC_URL```.

Also, make sure you've run ```source .env``` before using environment variables. You can test their availability by typing ```echo $VARIABLE_NAME``` in the command line.

**Reply by 0xmihirsahu** [reply]
Oh yeah thanks man. I figured it out

**Reply by EngrPips** [reply]
You can mark their help as the answer if it is what helped you figure it out.

**Comment by EngrPips** [comment]
Hello @0xmihirsahu. @thebraveman17 suggestions should help resolve your issue.
